solidity/dashboard/src/sagas/coverage-pool.js

```diff
@@ -153,7 +152,7 @@ export function* watchFetchCovPoolData() {
   )
 }
 
-export function* subscribeToCovTokenTransferEvent() {
+export function* subscribeToAssetPoolDepositedEvent() {
   const requestChan = yield actionChannel(
     COVERAGE_POOL_COV_TOKEN_TRANSFER_EVENT_EMITTED
```

Comment:
Can we also change this const name to `COVERAGE_POOL_ASSET_POOL_DEPOSITED_EVENT_EMITTED`?

Reply:
Please see 9129337